sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/MicroBatchExecution.scala

```diff
@@ -113,7 +114,9 @@ class MicroBatchExecution(
               v1.get.asInstanceOf[StreamingRelation].dataSource.createSource(metadataPath)
             nextSourceId += 1
             logInfo(s"Using Source [$source] from DataSourceV2 named '$srcName' $dsStr")
-            StreamingExecutionRelation(source, output)(sparkSession)
+            // We don't have a catalog table but may have a table identifier. Given this is about
```

Comment:
This is based on the fact we are encouraging 3rd party to use DSv2 for streaming data source whenever possible. (We don't make any improvements on DSv1 now.) When the data source implements DSv2, it is advised to implement the functionality of microbatch read.

Reply:
Actually this else statement is to handle edge case where the data source is based on DSv2 and implements continuous read but does not implement microbatch read.